src/Data/Poly/Internal/Sparse/Fractional.hs

```diff
@@ -58,6 +59,50 @@ quotientRemainder ts ys = case leading ys of
             zs = Poly $ G.singleton (xp - yp, xc / yc)
             xs' = xs - zs * ys
 
+-- | Execute the extended Euclidean algorithm.
+-- For polynomials 'a' and 'b', compute their unique greatest common divisor 'g'
```

Comment:
Single quotes in Haddock mean that the enclosed string is an identifier in scope. I suppose you meant to achieve monospace typesetting. In this case use `@a@`, `@b@`, etc.